<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QMainWindow" name="ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="headerMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Port</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hardware ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_scan"/>
   <addaction name="separator"/>
   <addaction name="action_connect"/>
   <addaction name="action_disconnect"/>
   <addaction name="separator"/>
   <addaction name="action_add"/>
   <addaction name="action_remove"/>
  </widget>
  <action name="action_scan">
   <property name="icon">
    <iconset>
     <normaloff>../Images/arrow_refresh.png</normaloff>../Images/arrow_refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Scan</string>
   </property>
   <property name="toolTip">
    <string>Scan available ports</string>
   </property>
  </action>
  <action name="action_connect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images/connect.png</normaloff>../Images/connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images/disconnect.png</normaloff>../Images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="action_add">
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="toolTip">
    <string>Add Connection</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove Connection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
